Support layout of rects and fix background css

chrisprice opened the pull request:
Fix for #318

ColinEberhardt commented:
LGTM - I added a few (pretty weak) unit tests. Also added a few comments, the various input arguments  for layout can get a bit confusing!

chrisprice commented:
Good call, thanks!

On Fri, 1 May 2015 22:48 Colin Eberhardt <notifications@github.com> wrote:

> LGTM - I added a few (pretty weak) unit tests. Also added a few comments,
> the various input arguments for layout can get a bit confusing!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3-financial-components/pull/323#issuecomment-98247973>
> .
>
